openapi.yml

```diff
@@ -325,16 +325,16 @@ components:
             given_name:
               type: string
           required:
-            - familyName
-            - givenName
+            - family_name
+            - given_name
         links:
           $ref: '#/components/schemas/Links'
         pecs:
           type: object
           additionalProperties:
             $ref: '#/components/schemas/EmailAddress'
       required:
-        - fiscal_Code
+        - fiscal_code
```

Comment:
The ontology name is tax_code. Spid uses fiscalNumber. This is a third one. Can we align to the ontology or there are some legacy issues for that?

Reply:
This is the same name used in all the other specs files of the IO project, for example for the [common functions](https://github.com/teamdigitale/io-functions-commons/blob/master/openapi/definitions.yaml) and for the [app backend](https://github.com/teamdigitale/io-backend/blob/master/api_backend.yaml). Currently, there is no issue changing this name, but there could be in the future when integration with other code is needed. I'd suggest to keep this name as it is, a proper story can be added to the pivotal dashboard as a reminder if necessary.

Reply:
Ok. I suggest the following:

-  advertise in some warning here and elsewhere that this label is not aligned with 
   the national schema for historical reasons and that it will be changed at some point
   in time.

Reply:
@ioggstream please don't add more entropy to this project, we have already enough.

We'll deal with the guidelines within other forums - besides `tax_code` became **after** the use of `fiscal_code` in this project, so if there is a "third one", it's `tax_code`.

Again, this is an **internal** API, it's not publicly available, it's not advertised anywhere, there's nobody else looking at this spec and asking why it's not aligned to your guidelines.

![standards](http://imgs.xkcd.com/comics/standards.png)